metrics/network/README.md

```diff
@@ -0,0 +1,87 @@
+# Networking tests for the Intel® Clear Containers runtime
```

Comment:
"the" is not necessary here. 
Should read: Networking tests for Intel® Clear Containers runtime